app/controllers/hub/tax_returns/certifications_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Hub
+  module TaxReturns
+    class CertificationsController < ApplicationController
+      include AccessControllable
+      before_action :require_sign_in
+      load_and_authorize_resource :tax_return, parent: false
+
+      def update
+        @tax_return.update(tax_return_params)
+        params[:tax_return_id] = nil
+        redirect_to referer_without_params || hub_client_path(id: @tax_return.client.id)
```

Comment:
Personally, I'd prefer to see an explicit `?next=...` URL passed in as a query string param.

Make sure it contains no `//` which could be used to switch host, and bob's your uncle, as they say in Australia.

Not a blocker; the referrer-based way is OK too.